index.html

```diff
@@ -12,10 +12,57 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
+        <div class="logo">
+          
+          <img class="logo" src="http://swarley00.github.io/startup-landing-page-clone/images/karma-logo.svg" alt="Logo Image">
+            
+        </div>
+        <nav class = "nav-bar">
+            <ul>
+                <li>Meet Karma</li>  <li>How-it-Works</li> 
+ <li>Store</li> 
+ <li>Blog</li>
+ <li>Help</li> 
+ <li>Login </li>   
+            </ul>
+                
 
+        </nav>
+      
+    </header>
+  <section class="introduction">
+    <div class= "intro-content">
+        <h1>Introducing Karma</h1>
+        <h3>Bring WiFi with you, everywhere you go.</h3>
+
+        <div class="btn">
```

Comment:
Use a `<button>` element here instead of a div - and you don't need the `<p>` inside.